presidio-analyzer/analyzer/app_tracer.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+
+class AppTracer:
+    """This class handles app traces"""
+    def __init__(self, enable_interpretability=True):
+
+        logger = logging.getLogger('Interpretability')
+        if not logger.handlers:
+            ch = logging.StreamHandler()
+            formatter = logging.Formatter(
+                '[%(asctime)s][%(name)s][%(levelname)s]%(message)s')
+            ch.setFormatter(formatter)
+            logger.addHandler(ch)
+            logger.setLevel(logging.INFO)
+            logger.propagate = False
+
+        self.logger = logger
+        self.enable_interpretability = enable_interpretability
+
+    def trace(self, request_id, trace_data):
+        """
+        Writes interpretability trace
+        :param request_id: A unique ID, to correlate across calls.
+        :param trace_data: A string to write.
+        :return:
+        """
```

Comment:
is there room for additional argument such as "trace_type"? currently trace is just a dump of all kinds of "logs" (result, artefacts, etc) and it may be hard to go through them if not typed in some way.

Reply:
This class accepts only string as input.
Are you suggesting other types?